server/src/test/java/com/hedera/block/server/consumer/ConsumerStreamResponseObserverTest.java

```diff
@@ -142,8 +145,10 @@ public void testResponseNotPermittedAfterCancel() {
                         testClock, streamMediator, serverCallStreamObserver, testContext);
 
         final List<BlockItem> blockItems = generateBlockItems(1);
+        final SubscribeStreamResponseSet subscribeStreamResponseSet =
```

Comment:
Just a note, the latest PR removes this message and replaces it with a more generic `BlockItemSet`, because it turned out we needed a `oneof` in the publish API too.